CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [0.18.8] - 2024-01-23
```

Comment:
version 0.18.7 has not been released yet.. please check properly next time

Reply:
Fixed